src/main/java/com/github/orest/car_shop/controllers/CarController.java

```diff
@@ -0,0 +1,114 @@
+package com.github.orest.car_shop.controllers;
+
+import com.github.orest.car_shop.model.Car;
+import com.github.orest.car_shop.repository.CarRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.view.RedirectView;
+
+import javax.validation.Valid;
+import java.util.Map;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+@RequestMapping
+@Controller
+public class CarController {
+
+    @Autowired
+    private CarRepository carRepository;
+
+    @GetMapping("/edit_car")
+    public String edit(@RequestParam int id, RedirectAttributes attributes, Model model) {
+
+        model.addAttribute("fields", Car.class.getDeclaredFields());
```

Comment:
You shouldn't have used Reflection API and `Model model`. Please read articles that I sent to you